lib/Service/FolderServiceV2.php

```diff
@@ -178,4 +178,21 @@ public function open(string $userId, ?int $folderId, bool $open): Entity
         $folder->setOpened($open);
         return $this->mapper->update($folder);
     }
+
+    /**
+     * Mark a folder as read
+     *
+     * @param string   $userId    Folder owner
+     * @param int      $id        Folder ID
+     * @param int|null $maxItemID Highest item ID to mark as read
+     *
+     * @throws ServiceConflictException
+     * @throws ServiceNotFoundException
+     */
+    public function read(string $userId, int $id, ?int $maxItemID = null): void
+    {
+        $folder = $this->find($userId, $id);
+
+        $this->mapper->read($userId, $folder->getId(), $maxItemID);
```

Comment:
why use `find()` with the user id and folder id, to just use the folder id from the returned entity?